### PR TITLE
Getting Started: update instructions + fix minor misspellings

### PR DESCRIPTION
**Pull request by stripes416**
Addresses changes & requests in #86 and #87 other than removing the prompt from the code examples (= PR #92)

### CODE REVIEW

**Comment by stripes416** [comment]
I know this doesn't address adding the script for users to run, but I wanted to make sure the other changes were good to go before writing a script based on the instructions.

> #86 @plbossart 
> In addition to the fix of the documentation, we should have a matching script that does everything for you. Instructions are always broken for months before people bother to report them, a broken script will trigger more reports. The hard-coded paths should go away, e.g. ~/work/sof should be replaced by a variable that can be easily adjusted. Thanks
> -Pierre

> #86 @intelkevinputnam 
> @plbossart I actually have a script that will pull all of the ".. code-block:: bash" sections from a .rst document and output to stdout. I agree about using variables, and I'd suggest starting the instructions for a series of steps by defining the variables that will be used later on in the instructions to make the resulting bash script easier to use.

File: getting_started/build-guide/build-from-scratch.rst [hunk 1]
```diff
@@ -252,72 +269,77 @@ CMake is designed for out-of-tree builds which is why you should make separate d
 
 You can manage builds for many configurations/platforms from the one source this way.
 
+.. note::
+
+   The *-j* argument indicites the number of cores to use in the build
```
Comment (intelkevinputnam) [inline getting_started/build-guide/build-from-scratch.rst]: Line: 274 "indicates"

File: getting_started/build-guide/build-from-scratch.rst [hunk 2]
```diff
@@ -25,31 +25,44 @@ Set up build environment
 
 Install package dependencies.
 
-.. code-block:: bash
+ * For Ubuntu 18.10:
```
Comment (intelkevinputnam) [inline getting_started/build-guide/build-from-scratch.rst]: Lines 28 and 38: There's an extra space here that is causing reStructuredText to interpret this as a block quote.

**Comment by stripes416** [comment]
@intelkevinputnam I've made those changes you requested. Thanks for reviewing my patches so consistently --- I really appreciate you catching things I don't see!

**Comment by intelkevinputnam** [comment]
@stripes416 My pleasure. :-)

**Pull request merged**